autotst/geometry.py

```diff
@@ -102,13 +102,20 @@ def __init__(
             atom_indices,
             dihedral,
             mask,
-            reaction_center=False):
+            angles,
+            description = 'hindered',
+            reaction_center = False,
+            center_atoms = []):
         self.index = index
         self.atom_indices = atom_indices
         self.dihedral = dihedral
         self.mask = mask
+        self.angles = angles
+        self.description = description.lower()
+        self.center_atoms = center_atoms
```

Comment:
This feels redundant because `center_atoms` are given in `atom_indices` as the 2nd and 3rd element of a 4 element list.

Reply:
Oh, I'm understanding this. This makes a lot more sense, maybe add a comment about this if that's the case then.